plotly/offline/offline.py

```diff
@@ -335,7 +335,33 @@ def iplot(figure_or_data, show_link=True, link_text='Export to plot.ly',
         figure_or_data, config, validate, '100%', 525, True
     )
 
-    ipython_display.display(ipython_display.HTML(plot_html))
+    figure = tools.return_figure_from_figure_or_data(figure_or_data, validate)
+
+    # Display 2 versions of the figure:
+    # - The raw HTML is the "old-school" style - we just throw in a JS script
+    #   inside the cell. This works for a lot of environments but some JS is
+    #   stripped out in some rendering environments / platforms like
+    #   GitHub, nteract, and eventually Plotly
```

Comment:
JavaScript is not stripped out in nteract - the core differences are that:

* `require` is commonjs require rather than requirejs' require in the desktop app
* we don't have the same JavaScript API as the Jupyter notebook

Neither of those should be relied on for HTML outputs regardless (or at least some primitive feature detection and fallbacks need to happen).